src/popup/components/AccountSwiper.vue

```diff
@@ -132,10 +132,15 @@ export default defineComponent({
 
     const swiper = computed(() => customSwiper.value?.$el.swiper);
 
-    async function setCurrentSlide(idx: number, slideParams?: number) {
+    async function setCurrentSlide(idx: number, slideSpeed?: number) {
       if (idx > -1 && currentIdx.value !== idx) {
         await nextTick();
-        swiper.value.slideTo(idx, slideParams);
+        await watchUntilTruthy(swiper);
+        const didSwipe = await swiper.value.slideTo(idx, slideSpeed);
+        if (!didSwipe) {
+          // Retry if slideTo fails
```

Comment:
When does this happen?

Reply:
This can happen while user is on multisig dashboard and active account is not one of the first ones (+3). If he refreshes the page it could happen that the initial slide to last active account will not work (and will return `false`)